src/components/SelectInput/SelectInput.jsx

```diff
@@ -154,18 +152,19 @@ const SelectInput = ({
     inputId: id,
     labelText: label,
     hasError: !!error,
+    className: 'm-bottom-xs',
   };
 
   return (
     <>
-      <div className={classNames('Palmetto-SelectInput', className, { disabled: isDisabled })}>
+      <div className={classNames('selectInputWrapper', className, { disabled: isDisabled })}>
```

Comment:
Should the standard be that each argument is on its own line?

Reply:
Let's start putting each argument on a new line. It looks more clean, and improves consistancy.